scripts/lib/upgrade-zulip-stage-2

```diff
@@ -145,13 +146,9 @@ if os.path.exists("/etc/init.d/postgresql"):
 
 if glob.glob("/usr/share/postgresql/*/extension/tsearch_extras.control"):
     # Remove legacy tsearch_extras package references
-    subprocess.check_call(
-        [
-            "su",
-            "postgres",
-            "-c",
-            'psql -v ON_ERROR_STOP=1 zulip -c "DROP EXTENSION IF EXISTS tsearch_extras;"',
-        ]
+    run_psql_as_postgres(
+        config_file=config_file,
+        sql_query="DROP EXTENSION IF EXISTS tsearch_extras;",
     )
```

Comment:
I think you can pull this back into the commit the introduced `run_psql_as_postgres`, which motivates that change a little better.